src/JSInterop/Microsoft.JSInterop.JS/src/src/Microsoft.JSInterop.ts

```diff
@@ -3,7 +3,7 @@
 export module DotNet {
   (window as any).DotNet = DotNet; // Ensure reachable from anywhere
 
-  export type JsonReviver = ((key: any, value: any) => any);
+  export type JsonReviver = ((key: any, value: any, byteArrays: Uint8Array[] | null) => any);
```

Comment:
If possible it would be preferable not to expose `byteArrays` to externally-registered revivers, since it's not intended to be part of the public API surface.

To do this it will probably be necessary to tweak the `parseJsonWithRevivers` logic so that it calls our own built-in reviver first in a hardcoded way with 3 params, then uses the `reduce` logic with 2 params for all externally-registered ones.